### ISSUE TITLE
Convert Editor's Note to normal notes

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/respec#1239]
... during processing, Editor's Note should become normal notes. The we can remove any CSS that's related to Editor's Note.

**Comment by OrionStar25** [comment]
I'd like to try solving this please :3

**Comment by Swapnilr1** [comment]
If no one is working on this, can I take it up? Also, if possible, can anyone explain this issue in some detail?

**Comment by marcoscaceres** [comment]
Ok, so the idea with this one is you can do:

```
<div class="note" title="Hi, I'm a note!">
  <p>
     This is a note. 
  </p>
</div> 

<!-- ed note --> 
<div class="ednote" title="Hi, I'm a note!">
  <p>
     This is an editor note. 
  </p>
</div>

<div class="issue" title="Hi, I'm an issue!">
  <p>
     This is an issue. 
  </p>
</div>
```

You will notice they look the same (except issues are red).

There are few small, but non-trivial, tasks here:

 1. Look in ReSpec's code for "ednote" - and figure out what it's doing. 
 1. Look at https://github.com/w3c/respec/blob/develop/js/core/css/issues-notes.css for "ednote" 
 1. Look at https://github.com/w3c/tr-design/blob/gh-pages/src/base.css and see how much overlap there is between issues-notes.css and base.css.
 1. Remove redundant CSS from issues-notes.css relate to notes, ednotes, and possibly issues. 

Basically, if it's in base.css, then it doesn't need to be in ReSpec's css.

**Comment by Swapnilr1** [comment]
Thanks. It seems to me I have understood what needs to be done. I will begin work after I finish #2005

**Comment by Swapnilr1** [comment]
I just want to confirm if I have understood this issue correctly. 
Since editor's notes and normal notes look same, the same CSS can be applied to them. Now https://github.com/w3c/tr-design/blob/gh-pages/src/base.css does not have CSS for `.ednote` class, so when ReSpec runs on the HTML document, it should change all `ednote` classes to `note`. 
And along with it, I also remove redundant CSS code. 

Is it correct?

**Comment by marcoscaceres** [comment]
@Swapnilr1 yes, what you describe is correct.

**Comment by Swapnilr1** [comment]
@marcoscaceres One more question. For editor's notes, the text "Editor's note: " is added in the title, while for a normal note, the text "Note: " is added in the title. Do we wish to do away with editor's notes entirely (so even editor's notes would only have ""Note: " added in the title) or just wish to use common CSS?

**Comment by marcoscaceres** [comment]
> Do we wish to do away with editor's notes entirely (so even editor's notes would only have "Note: " added in the title) or just wish to use common CSS?

Ah, good point about the title. I think we should just use common CSS.
